boxsdk/session/session.py

```diff
@@ -184,6 +186,14 @@ def client_config(self):
         """
         return self._client_config
 
+    @property
+    def network_config(self):
+        """
+
+        :rtype:     :class:`Network`
```

Comment:
It's a bit weird that this config class has a name conflict with the `Network` interface that `DefaultNetwork` implements — we may want to disambiguate them